src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,145 +1,177 @@
 import edu.wpi.first.wpilibj2.command.button.Trigger;
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.CommandBase;
-import edu.wpi.first.wpilibj2.command.CommandScheduler;
-
-
-import java.io.File;
-import java.util.Scanner;
-
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 
-import edu.wpi.first.wpilibj.Filesystem;
-
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+import java.util.function.DoubleSupplier;
 
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.IntakeSubsystem.SwitchIntakeDirection;
+import frc.robot.subsystems.TankDriveSubsystem.SwitchDrivetrainInvert;
+import frc.robot.commands.IntakeCommand;
 
 /**
- * This class is where the bulk of the robot should be declared.  Since Command-based is a
- * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
- * periodic methods (other than the scheduler calls).  Instead, the structure of the robot
- * (including subsystems, commands, and button mappings) should be declared here.
+ * This class is where the bulk of the robot should be declared. Since
+ * Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in
+ * the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of
+ * the robot
+ * (including subsystems, commands, and button mappings) should be declared
+ * here.
  */
 public class RobotContainer {
-
   private static RobotContainer m_robotContainer = new RobotContainer();
-
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-// Subsystems
+  // Subsystems
   public final TankDriveSubsystem m_tankDriveSubsystem = new TankDriveSubsystem();
-  public final VisionSubsystem m_VisionSubsystem = new VisionSubsystem();
+  public final IntakeSubsystem m_intake = new IntakeSubsystem();
+  private static MiscDashboardSubsystem m_miscDashboardSubsystem = new MiscDashboardSubsystem();
+
+  // other stuff
+  private final CommandXboxController m_driverHID = new CommandXboxController(0);
+  private final CommandPS4Controller m_operatorHID = new CommandPS4Controller(1);
+
+  // Commands
+  private DoubleSupplier leftDriveTrain = () -> m_driverHID.getRawAxis(1);
+  private DoubleSupplier rightDriveTrain = () -> m_driverHID.getRawAxis(5);
 
-// Joysticks
-  private final CommandGenericHID m_driverHID = new CommandGenericHID(0);
-  
+  private final TankDriveSubsystem.driveMotorCommand m_driveCommand = m_tankDriveSubsystem.new driveMotorCommand(
+      rightDriveTrain,
+      leftDriveTrain,
+      m_tankDriveSubsystem);
+
+  private final TankDriveSubsystem.driveMotorCommand m_operatorDrive = m_tankDriveSubsystem.new driveMotorCommand(
+      () -> m_operatorHID.getRawAxis(1),
+      () -> m_operatorHID.getRawAxis(5),
+      m_tankDriveSubsystem);
```

Comment:
I believe @Benito-Esposti / @tobyMickelson decided that giving operator the ability to drive would be more dangerous or confusing than useful. If true, we should remove this code as well as the switch enabling it down around like 99.

Reply:
Operator drive? _**WHAT OPERATOR DRIVE??**_